src/renderer.js

```diff
@@ -86,13 +94,9 @@ Renderer.prototype.render = function () {
   gl.viewport(0, 0, gl.drawingBufferWidth, gl.drawingBufferHeight);
   gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT);
 
-  var uniforms = updateUniforms(gl, this.camera, this.uniforms);
-
-  //renderPass(gl, depthProgramInfo, faceBufferInfo, uniforms, 'TRIANGLES');
+  var uniforms = updateUniforms(gl, this.camera, this.glTexture, this.uniforms);
   renderPass(gl, this.shaders.face, this.buffers.face, uniforms, 'TRIANGLES');
-  //renderPass(gl, this.shaders.fold, this.foldBufferInfo, uniforms, 'LINES');
 
-  //renderPoints(points, pointDivs);
   this.raf = requestAnimationFrame(this.render);
```

Comment:
Unconditionally requesting a new frame might be resource intensive on less capable hardware. =P Ideally there would be a check here to see if a new frame is actually needed and render calls in places that would cause it to start up again (e.g. resize). No action necessary in this revision.